src/LSSOLQPSolver.h

```diff
@@ -69,6 +69,8 @@ class TASKS_DLLAPI LSSOLQPSolver : public GenQPSolver
 	Eigen::VectorXd C_;
 
 	int nrALines_;
+public:
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
just being conservative on this... I assumed no unfortunately but haven't verified.